blocks/scroll/scroll.css

```diff
@@ -0,0 +1,16 @@
+@import url('./__button-container/scroll__button-container.css');
+@import url('./__button/scroll__button.css');
+@import url('./__button-text/scroll__button-text.css');
+@import url('./__button/_next/scroll__button_next.css');
+@import url('./__button/_prev/scroll__button_prev.css');
+@import url('./__dots/scroll__dots.css');
+@import url('./__dot/scroll__dot.css');
+@import url('./__dot/_active/scroll__dot_active.css');
```

Comment:
It's better to import element modifiers inside element file. This change makes your styles more encapsulated

Reply:
I moved the scroll__button_prev and scroll__button_next modifiers inside the element file.